types/compilation/compilation.interfaces.ts

```diff
@@ -59,7 +59,7 @@ export type CompilationResult = {
     downloads?: BuildEnvDownloadInfo[];
     gccDumpOutput?: any;
     languageId?: string;
-
+    source?: string;
```

Comment:
this looks like a bug, in what situation is this used?

Reply:
Found the issue, will make a separate bugreport for it. You don't have to fix it.